### ISSUE TITLE
error stack names with versions

### ISSUE THREAD

**Issue by saitodisse** [azukiapp/azk-dev#2]
Would be nice if `azk-dev` could generate the `azk_gulp.sourcemaps_path` property automatically based on the name of the package and its version. To be even better, it could generate with a different configurable color. Thats what I did in `castborg` and `logmycode`.

**gulpfile.js**

``` js
require('source-map-support').install();

var chalk = require('chalk');
var package_json = require('./package.json');
var stack_trace_name = [
  '/',
  chalk.styles.gray.close,
  chalk.bold.green(package_json.name),
  chalk.gray('('),
  chalk.green(package_json.version),
  chalk.gray(')'),
  chalk.styles.gray.open,
].join('');

var azk_gulp = require('azk-dev/gulp')({
  cwd  : __dirname,
  sourcemaps_path: stack_trace_name,
  lint: [ "bin/**/*.js" ], // Extra files for the lint analyzer
  default: ['watch:test:lint'],
});

var gulp = azk_gulp.gulp;

gulp.task("show:args", "Help text", ["before:show"], function() {
  console.log(azk_gulp.yargs.argv);
  return null;
}, { aliases: ["sa", "s"] });
```

![image](https://cloud.githubusercontent.com/assets/155953/7265447/98cd51fe-e870-11e4-9fad-d358ae59cc5b.png)

**Comment by nuxlli** [comment]
I liked the suggestion, but has a problem: Files generated thus carry a sequence of escaped characters, which makes them work well where colors are supported. But end up strangers where it is not supported or stacktracers browsers for example.

**Comment by gullitmiranda** [comment]
Another suggestion,
Add a space after the "/" after the project name, thus enabling click to open the editor from the terminal.

eg:

``` shell
# before:
ReferenceError: er is not defined
    at cli (/azk:0.16.2/src/cli/index.js:65:15)

# after:
ReferenceError: er is not defined
    at cli (azk(0.16.2) src/cli/index.js:65:15)
```

**Comment by nuxlli** [comment]
Closes with 42c94cce42497
